grakn-core/src/main/java/ai/grakn/util/Schema.java

```diff
@@ -133,7 +134,8 @@ public static boolean isMetaLabel(Label label) {
         ATTRIBUTE(Attribute.class),
 
         //Internal
-        SHARD(Vertex.class);
+        SHARD(Vertex.class),
+        CONCEPT(Concept.class);
```

Comment:
New base type? :O

Reply:
It's used in the link class so I don't have to hard code the string "concept" there. The other way would be to have more Paths on `REST.WebPath` which would be redundant. This feels like the least evil appraoch.

Reply:
That's fair! I'd definitely have a comment to say that no vertex will _really_ have this type.